docs/developer-docs/gas-cost.md

```diff
@@ -1,100 +1,212 @@
-| Ingress Byte Reception               | For every byte received in an ingress message                                                                    | Receiving canister | $0.00000000261772           | $0.00000000684534           |
-| GB Storage Per Second                | For storing a GB of data per second                                                                              | Canister with storage | $0.00000016622522           | $0.00000043474178           |
- |                                      |                                                                                                                  |                             |                             |
- | *HTTPS outcalls*                     |                                                                                                                  |                             |                             |
- | HTTPS outcall (per call)                | For sending an HTTPS outcall to a server outside ICP, per message (`http_request`)                            | Sending canister | $0.0000643173804               | $0.0002242862496 |
- | HTTPS outcall request message size (per byte)	| For sending an HTTPS outcall to a server outside ICP, per request byte (http_request)	| Sending canister | $0.000000006806072 |	$0.000000017800496 |
-| HTTPS outcall response message size (per byte)	| For sending an HTTPS outcall to a server outside ICP, per reserved response byte (http_request) | Sending canister | $0.000000013612144	| $0.000000035600992 |
+![Canister calls overview](./_attachments/transaction-overview.png)
 
-Cost per Transaction in USD (XDR/USD exchange rate as of November 23, 2022).
+Each type of canister operation has a different cycles cost associated with it, and the canister responsible for paying the cycles varies based on the type of operation.
 
-The following table shows the calculated storage cost per GB for a 30-day month: 
+### Canister creation
 
-|                      |                                    | 13-node Application Subnets | 34-node Application Subnets |
-|----------------------|------------------------------------|-----------------------------|-----------------------------|
-| GB Storage Per Month | For storing a GB of data per month | $0.431                      | $1.127                      |
+Canister creation costs 100B cycles or approximately $0.133661 USD. Canisters can be created by users or other canisters.
+
+### Messaging
+
+A canister can receive messages from users and other canisters. In canister-to-canister messages, the sending canister pays message transmission costs. In user-to-canister messages, the receiving canister covers the message transmission costs (see [reverse gas model](#the-icp-reverse-gas-model). User-to-canister messages are also referred to as ingress messages.
```

Comment:
There is one closing `)` bracket missing in "(see ...".

Reply:
Fixed missing )